src/OpenTelemetry/Trace/Configuration/SpanProcessorPipelineBuilder.cs

```diff
@@ -67,6 +69,16 @@ public SpanProcessorPipelineBuilder SetExportingProcessor(Func<SpanExporter, Spa
             return this;
         }
 
+        /// <summary>
+        /// Configures exporter.
+        /// </summary>
+        /// <param name="exporter">Factory that creates the exporter instance.</param>
+        public SpanProcessorPipelineBuilder SetExporter(Func<Resource, SpanExporter> exporter)
```

Comment:
resource should not be part of exporter initialization, it's passed in each span and populated by tracer